test/test_core.py

```diff
@@ -5779,6 +5779,15 @@ def test_fs_nodefs_readdir(self):
     self.emcc_args += ['-lnodefs.js']
     self.do_runf('fs/test_nodefs_readdir.c', 'success')
 
+  @requires_node
```

Comment:
Can you mark this tests as `@crossplatform`?

Reply:
test/core/test_statvfs.out